log-viewer/modules/TreeView.ts

```diff
@@ -127,7 +127,7 @@ type OpenInfo = {
 };
 
 function openMethodSource(info: OpenInfo) {
-  if (info) {
+  if (info && window.vscodeAPIInstance) {
     window.vscodeAPIInstance.postMessage(info);
```

Comment:
Would be nice if we injected the vscodeAPIInstance. 
Or maybe we could send a message that the vscode plugin side listens for and forwards to the vscode instance.

Reply:
Agree, I did something similar on dep. explorer to abstract postMessage a bit